server.go

```diff
@@ -211,10 +289,10 @@ func Serve(opts *ServeConfig) {
 	// Output the address and service name to stdout so that core can bring it up.
 	fmt.Printf("%d|%d|%s|%s|%s%s\n",
 		CoreProtocolVersion,
-		opts.ProtocolVersion,
+		protoVersion,
```

Comment:
Not really related to what you're doing here, but I notice the comment above refers to "core", which I guess was probably originally in reference to "Packer Core" or "Terraform Core" (depending on the heritage of this code) but should probably now say "the client". :grinning: